### PR TITLE
chore: remove unnecessary log, using debug instead

### PR DESCRIPTION
**Pull request by gxcsoccer**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: lib/leader.js [hunk 1]
```diff
@@ -172,7 +173,7 @@ class Leader extends Base {
   _doSubscribe(reg, conn) {
     const key = this.options.formatKey(reg);
     const callback = err => {
-      if (err) {
+      if (err && !this._closeByUser) {
```
Comment (gxcsoccer) [inline lib/leader.js]: leader 给 follower 发数据的时候调用 close，避免不必要的 error log

**Comment by gxcsoccer** [comment]
很多日志没必要打，都改成 debug

**Comment by fengmk2** [comment]
ci 重跑，成功后合并。

**Comment by fengmk2** [comment]
1.6.4

**Pull request merged**